index.bs

```diff
@@ -136,22 +136,26 @@ interface DetectedFace {
 
 <pre class="idl">
 dictionary Landmark {
-  required Point2D location;
+  required FrozenArray&lt;Point2D> location;
   LandmarkType type;
 };
 </pre>
 
 <dl class="domintro">
   <dt><dfn dict-member for="Landmark"><code>location</code></dfn></dt>
   <dd>Location of a given landmark aligned to the image axes.</dd>
+  <dd>A point or a <a>sequence</a> of points of the detected landmark.</dd>
   <dt><dfn dict-member for="Landmark"><code>type</code></dfn></dt>
   <dd>Type of the landmark, if known.</dd>
 </dl>
 
 <pre class="idl">
 enum LandmarkType {
   "mouth",
-  "eye"
+  "eye",
+  "brow",
+  "pupil",
```

Comment:
At a minimum, at least 2 platforms should support the new landmarks, but I only see "nose" as available in [Google Play Services](https://developers.google.com/android/reference/com/google/android/gms/vision/face/Landmark), can you see the eye brow and/or pupils? If not, I'd recommend against including them at this point.